PhpAmqpLib/Helper/MiscHelper.php

```diff
@@ -77,7 +77,7 @@ public static function hexdump($data, $htmloutput = true, $uppercase = false, $r
         // Iterate string
         for ($i = $j = 0; $i < $len; $i++) {
             // Convert to hexidecimal
-            $hexi .= sprintf('%02$x ', ord($data[$i]));
+            $hexi .= sprintf("%02$x ", ord($data[$i]));
```

Comment:
How do you know if this wasn't itentional? Would you mind creating a test case, so if this really is a bug, nobody will break it again?